hie-base/Haskell/Ide/Engine/PluginTypes.hs

```diff
@@ -18,8 +18,14 @@ module Haskell.Ide.Engine.PluginTypes
   , ParamValP(..)
   , ParamMap
   , pattern ParamTextP
+  , pattern ParamIntP
+  , pattern ParamBoolP
   , pattern ParamFileP
```

Comment:
Fair enough, I just wonder if we need multiple position and file types at all. I could go either way, but simplicity is good.